backend/src/cache/server/mutation.go

```diff
@@ -119,14 +119,43 @@ func MutatePodIfCached(req *v1beta1.AdmissionRequest, clientMgr ClientManagerInt
 
 	annotations[ExecutionKey] = executionHashKey
 	labels[CacheIDLabelKey] = ""
-	var maxCacheStalenessInSeconds int64 = -1
-	maxCacheStaleness, exists := annotations[MaxCacheStalenessKey]
+
+	var cacheStalenessInSeconds int64 = -1
+	var userCacheStalenessInSeconds int64 = -1
+	var defaultCacheStalenessInSeconds int64 = -1
+	var maximumCacheStalenessInSeconds int64 = -1
+
+	userCacheStaleness, exists := annotations[MaxCacheStalenessKey]
 	if exists {
-		maxCacheStalenessInSeconds = getMaxCacheStaleness(maxCacheStaleness)
+		log.Printf("userCacheStaleness: %s", userCacheStaleness)
+		userCacheStalenessInSeconds = stalenessToSeconds(userCacheStaleness)
+		log.Printf("userCacheStalenessInSeconds: %d", userCacheStalenessInSeconds)
+	}
+	defaultCacheStaleness, exists := os.LookupEnv("DEFAULT_CACHE_STALENESS")
+	if exists {
+		log.Printf("defaultCacheStaleness: %s", defaultCacheStaleness)
+		defaultCacheStalenessInSeconds = stalenessToSeconds(defaultCacheStaleness)
+		log.Printf("defaultCacheStalenessInSeconds: %d", defaultCacheStalenessInSeconds)
```

Comment:
ditto: excessive logging, and same for down below

Reply:
I have removed it as demanded